docs/core/additional-tools/dotnet-svcutil-guide.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Microsoft WCF dotnet-svcutil Tool
+description: An overview of the Microsoft WCF dotnet-svcutil Tool that adds functionality for .NET Core and ASP.NET Core projects, similar to the WCF svcutil tool for .NET Framework projects.
+author: mlacouture
+ms.author: mlacouture
+ms.date: 05/31/2018
+---
+# Microsoft WCF dotnet-svcutil Tool
+
+The Windows Communication Foundation (WCF) **dotnet-svcutil** tool is a .NET Core CLI tool that retrieves metadata from a web service on a network location or from a WSDL file, and generates a WCF class containing client proxy methods that access the web service operations.
+
+Similar to the [**Service Model Metadata - svcutil**](/dotnet/framework/wcf/servicemodel-metadata-utility-tool-svcutil-exe) tool for .NET Framework projects, the **dotnet-svcutil** is a command line tool for generating a web service reference compatible with .NET Core and .NET Standard projects.
```

Comment:
change link to ../../framework/wcf/servicemodel-metadata-utility-tool-svcutil-exe.md

command line tool -> command-line tool